state/mongodb/mongodb.go

```diff
@@ -51,6 +52,8 @@ const (
 	id               = "_id"
 	value            = "value"
 	etag             = "_etag"
+	daprttl          = "_dapr_ttl"
```

Comment:
Any reason why this attribute is prefixed with dapr while the others don't?

Reply:
I would like to have seen the others prefixed as well, but that cannot be changed now without a proper migration strategy.

Reply:
IMHO because the collection is not meant for accessing outside of Dapr, so the prefix isn't needed. But I'm fine with it

Reply:
Let's remove the prefix while we can since its a bit of a eye sore.